modules/tensor_mechanics/tests/initial_stress/tests

```diff
@@ -4,4 +4,12 @@
     input = 'gravity.i'
     exodiff = 'gravity.e'
   [../]
+  [./mc_tensile]
+    type = 'CSVDiff'
+    input = 'mc_tensile.i'
+    csvdiff = 'mc_tensile.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    cli_args = '--no-trap-fpe'
```

Comment:
Yes, @permcody , in debug mode i always hit FPEs.  This is because of the PETSc-LAPACK singular value decomposition.  Before doing stuff, it checks that 0/0 and 1/0 behave as expected, and that trips a FPE.  In opt mode everything is fine.